### PR TITLE
Changelog for 1.11.0

### PR DESCRIPTION
**Pull request by straight-shoota**
Preview: https://github.com/straight-shoota/crystal/blob/changelog/1.11.0/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,245 @@
...
+- *(interpreter)* Support `-dynamic.lib` in Windows interpreter ([#14143](https://github.com/crystal-lang/crystal/pull/14143), thanks @HertzDevil)
+- *(interpreter)* Support absolute paths in `CRYSTAL_INTERPRETER_LOADER_INFO` ([#14147](https://github.com/crystal-lang/crystal/pull/14147), thanks @HertzDevil)
+- *(interpreter)* Add `Crystal::Repl#parse_and_interpret` ([#14138](https://github.com/crystal-lang/crystal/pull/14138), thanks @bcardiff)
+- *(semantic)* Change short_reference for top-level methods to `::foo` ([#14071](https://github.com/crystal-lang/crystal/pull/14071), thanks @keshavbiswa)
+
+#### tools
+
+- *(docs-generator)* Expose inherited macros in generated API docs ([#13810](https://github.com/crystal-lang/crystal/pull/13810), thanks @Blacksmoke16)
+- *(docs-generator)* Order macros below class methods in generated docs ([#14024](https://github.com/crystal-lang/crystal/pull/14024), thanks @Blacksmoke16)
+- *(formatter)* Do not remove trailing comma from multi-line macro/def parameters ([#14075](https://github.com/crystal-lang/crystal/pull/14075), thanks @Blacksmoke16)
```
Comment (Blacksmoke16) [inline CHANGELOG.md:73]: I know we have https://github.com/crystal-lang/crystal/issues/13002, but are we really thinking of waiting until _all_ of those issues are completed before making these changes available? Or if not, what constitutes a big enough batch?

Comment (straight-shoota) [reply CHANGELOG.md:73]: Let's discuss that in #13002, shall we?
It's a good point that we should mark this change as not yet active, though.

Comment (Blacksmoke16) [reply CHANGELOG.md]: Worth backporting the same thing to the other `&` to yielding methods change from a few versions back?

**Review by beta-ziliani — COMMENTED** [review]
Two minor fixes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,257 @@
...
+
+### Bugfixes
+
+#### stdlib
+
+- Fix `Box(T?)` crashing on `nil` ([#13893](https://github.com/crystal-lang/crystal/pull/13893), thanks @HertzDevil)
+- Fix typos in src ([#14053](https://github.com/crystal-lang/crystal/pull/14053), thanks @kojix2)
+- *(collection)* Fix `Indexable#each_repeated_combination(n)` when `n > size` ([#14092](https://github.com/crystal-lang/crystal/pull/14092), thanks @HertzDevil)
+- *(concurrency)* Make `Process#wait` asynchronous on Windows ([#13908](https://github.com/crystal-lang/crystal/pull/13908), thanks @HertzDevil)
+- *(concurrency)* MT: math overflow after spawning Int32::MAX + 1 fibers ([#14096](https://github.com/crystal-lang/crystal/pull/14096), thanks @ysbaddaden)
```
[the remark this answers is not in the document]
Comment (straight-shoota) [reply CHANGELOG.md:87]: We wouldn't apply these changes here manually. Instead, rename the respective PRs directly The next regeneration of the changelog will reflect those changes.

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**